relay-dynamic-config/src/defaults.rs

```diff
@@ -25,12 +28,77 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
         return;
     }
 
+    // Add conditions to filter spans if a specific module is enabled.
+    // By default, this will extract all spans.
+    let mut span_op_conditions: Vec<RuleCondition> = Vec::new();
+    let span_op_field_name = "span.op";
+
+    if project_config
+        .features
+        .has(Feature::SpanMetricsExtractionAllModules)
+    {
+        span_op_conditions.push(RuleCondition::And(AndCondition { inner: Vec::new() }));
+    } else {
+        if project_config
+            .features
+            .has(Feature::SpanMetricsExtractionDBModule)
+        {
+            span_op_conditions.push(RuleCondition::And(AndCondition {
+                inner: vec![
+                    RuleCondition::Glob(GlobCondition {
+                        name: span_op_field_name.into(),
+                        value: GlobPatterns::new(vec!["db*".into()]),
+                    }),
+                    RuleCondition::Not(NotCondition {
+                        inner: Box::new(RuleCondition::Eq(EqCondition {
+                            name: span_op_field_name.into(),
+                            value: Value::Array(vec![
+                                "db.clickhouse".into(),
+                                "db.redis".into(),
+                                "db.sql.query".into(),
+                            ]),
+                            options: Default::default(),
+                        })),
+                    }),
+                ],
+            }));
+        }
+
+        if project_config
+            .features
+            .has(Feature::SpanMetricsExtractionBrowserModule)
+        {
+            span_op_conditions.push(RuleCondition::Glob(GlobCondition {
+                name: span_op_field_name.into(),
+                value: GlobPatterns::new(vec![
+                    "browser*".into(),
+                    "http*".into(),
+                    "resource".into(),
```

Comment:
`resource` can also be `resource.script`, and `resource.css` so I think we have to glob here as well.